commcare_connect/organization/decorators.py

```diff
@@ -16,6 +18,10 @@ def _request_user_is_admin(request):
     ) or request.user.is_superuser
 
 
+def _request_user_is_viewer(request):
+    return (request.org and request.org_membership) or request.user.is_superuser
```

Comment:
I would have expected this to check on `request.org_membership.is_viewer`

Reply:
There is a slight caveat here in naming, the viewer access is allowed for all roles i.e. MEMBER, ADMIN, or VIEWER. Hence this doesn't check if a user is a viewer.